droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/action/CleanCommand.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2016, The National Archives <pronom@nationalarchives.gov.uk>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following
+ * conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ *
+ *  * Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ *  * Neither the name of the The National Archives nor the
+ *    names of its contributors may be used to endorse or promote products
+ *    derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package uk.gov.nationalarchives.droid.command.action;
+
+import org.apache.commons.io.FileUtils;
+
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
+public class CleanCommand implements DroidCommand {
+
+    private final Path droidHome;
+    private final Stream<String> toDelete;
+
+    public CleanCommand(final Path droidHome) {
+        this.droidHome = droidHome;
+        this.toDelete = Stream.of(
+                "container_sigs",
+                "signature_files",
+                "profiles",
+                "profile_templates",
+                "droid.properties",
+                "log4j2.properties",
+                "tmp"
+        );
+    }
+
+    @Override
+    public void execute() throws CommandExecutionException {
```

Comment:
Should this maybe return a confirmation message (that tells them that files are deleted and potentially shows them what's in the directory now) or is the fact that it didn't throw an error, enough?

Reply:
This overrides a method from the parent `DroidCommand` class so it has to be `void execute()` so we can't return anything.